x-pack/plugins/infra/public/components/asset_details/tabs/profiling/profiling_links.tsx

```diff
@@ -43,7 +44,12 @@ export function ProfilingLinks({
         </EuiLink>
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
-        <EuiLink data-test-subj="infraFlamegraphTestLink" href={PROFILING_FEEDBACK_URL} external>
+        <EuiLink
+          data-test-subj="infraFlamegraphTestLink"
+          href={PROFILING_FEEDBACK_URL}
+          external
+          target="_blank"
```

Comment:
Has something changed in EUI? Last time I checked, the `external` prop was also adding `target="_blank"`.

Reply:
Not sure, but when I tested it, it was opening on the same tab, AFAICT the `external` property only adds an icon to the link.

Reply:
Got it 👍 btw, `external` also adds the `rel="noopener noreferrer"`